ci-operator/step-registry/ibmcloud/provision/dedicatedhost/static/ibmcloud-provision-dedicatedhost-static-commands.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+#pre create the dedicated hosts based on the ${SHARED_DIR}/dedicated_host.yaml and update "${SHARED_DIR}/dedicated_host.yaml
+#save the dedicated host name for master nodes in "${SHARED_DIR}/dedicated_host"
+#save the dedicated host name for worker nodes in "${SHARED_DIR}/dedicated_host" 
```

Comment:
The comments need some update, right?

Reply:
yes. I will update when current test done.